lib/config/api_config.dart

```diff
@@ -0,0 +1,4 @@
+class ApiConfig {
```

Comment:
What's the idea behind having an ApiConfig if all the API routes are already defined in `Routes.dart`?

Reply:
Yes make sense.
Done ✅